### PR TITLE
Support to display total install size in statistics view

### PR DESCRIPTION
**Pull request by darklinkpower**
I have verified that:
- [x] These changes work, by building the application and testing them.
- [x] Pull request is targeting `devel` branch.
- [x] I added myself into [contributors file](https://github.com/JosefNemec/Playnite/blob/devel/source/Playnite.DesktopApp/Resources/contributors.txt) if I want to receive contribution credit in the application itself.

### Screenshots

image

### CODE REVIEW

**Review by ForAnAngel731 — APPROVED** [review]

**Comment by ForAnAngel731** [comment]
Can you add the option to sort by installation size in the library view?

File: source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs [hunk 1]
```diff
@@ -169,6 +169,7 @@ public class GameStats
             public BaseStatInfo Favorite { get; set; }
             public ulong TotalPlayTime { get; set; }
             public ulong AvaragePlayTime { get; set; }
+            public ulong? TotalInstallSize { get; set; }
```
[the remark this answers is not in the document]
Comment (darklinkpower) [reply source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs]: Done in https://github.com/JosefNemec/Playnite/pull/3327/commits/f10de22246dba0984bd6f0afd60736501d4f85c0

File: source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs [hunk 2]
```diff
@@ -321,6 +322,7 @@ private GameStats FillData(bool filtered)
             ulong hidden = 0;
             ulong favorite = 0;
             ulong totalPlaytime = 0;
+            ulong? totalInstallSize = 0;
```
Comment (JosefNemec) [inline source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs:325]: You forgot about this one, doesn't need to be nullable either.

Comment (darklinkpower) [reply source/Playnite.DesktopApp/ViewModels/StatisticsViewModel.cs]: Done in https://github.com/JosefNemec/Playnite/pull/3327/commits/ea0df032562d4678b175973b4e8bfaee7a6cd58a

**Review by JosefNemec — APPROVED** [review]

**Pull request merged**